open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    infected.loc['US'][2:].plot()
</pre><p>You can create a function as <code>plot_infected_vs_recovered()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>DONE</p>